subprojects/signing/src/integTest/groovy/org/gradle/plugins/signing/SigningConfigurationsIntegrationSpec.groovy

```diff
@@ -51,6 +51,35 @@ class SigningConfigurationsIntegrationSpec extends SigningIntegrationSpec {
         file("build", "libs", "sign-1.0-sources.jar.asc").text
     }
 
+    @ToBeFixedForConfigurationCache
+    def "signing configurations is idempotent"() {
+        given:
+        buildFile << """
+            configurations {
+                meta
+            }
+
+            signing {
+                ${signingConfiguration()}
+                sign configurations.archives, configurations.meta, configurations.archives, configurations.meta
```

Comment:
Pls. make this consistent with the other test change.

Reply:
Changed the test to be consistent.